Engine/Source/Runtime/ECWorld/AnimationComponent.h

```diff
@@ -16,6 +16,17 @@ class Track;
 namespace engine
 {
 
+enum class AnimationClip
+{
+	Idel,
+	Walking,
+	Running,
+
+	Blend,
+	Switch,
+	Count,
```

Comment:
No need to use `Count`. You can get it by using nameof :
auto enumCount = nameof::enum_count<EnumType>();